build/container-test-scripts/integration/fedora/dotnet-install.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -o xtrace
+set -e
+
+# See https://github.com/OctopusDeploy/BuildAgentAutomation/blob/main/Common/shared/dotnet-install.sh
+curl --silent --fail -L -O https://dot.net/v1/dotnet-install.sh || exit 1
+chmod +x dotnet-install.sh || exit 1
+
+./dotnet-install.sh --channel 6.0
```

Comment:
Wondering if this would be easier to update/find when trying to update if it were sourced from a build environment variable? Do we do something similar elsewhere?

Reply:
It could even go in our script to set dotnet env vars - that feels discoverable

Reply:
We currently set the version of tentacle in its csproj file, I would opt for keeping as much as we can in git rather than spread across git and teamcity.

Reply:
I don't know if setting the channel in the dotnet env vars file would be more discoverable.

Reply:
Then again on the other hand are we already setting 6.0 as a env var that would need to be updated anyway...

Reply:
had a look and could not see one that we would hook into. We set agent parameters requirements  e.g. netcore must contain 6.0. But we don't actually get given net6